src/write_concern.ts

```diff
@@ -65,7 +66,8 @@ export class WriteConcern {
   readonly w?: W;
   /** Request acknowledgment that the write operation has been written to the on-disk journal */
   readonly journal?: boolean;
-  /** Specify a time limit to prevent write operations from blocking indefinitely */
+  /** Specify a time limit to prevent write operations from blocking indefinitely
+   * @deprecated Will be removed in the next major version. Please use timeoutMS */
```

Comment:
Let's use the `expectDeprecated` helper in tsd to assert none of these get undeprecated and it will help us find them all later. It helps if we refactor the way we declare the options but we preserve these docs and info.